### PR TITLE
Bugfix: Update the correct scope

### PR DESCRIPTION
**Pull request by michielgerritsen**
Thank you for creating this pull request! To make the best use of your and our time we created this checklist to get the best possible pull requests:

- [ ] The code is working on a plain Magento 2 installation.
- [ ] The code follows the PSR-2 code style.
- [ ] When an exception or error is logged the message is accompanied with some context, eg: `Error when trying to get the payment status:`
- [ ] Contains tests for the changed/added code (great if so but not required).
- [ ] I have added a scenario to test my changes.

**This PR touches code in the following areas (Check what is applicable):**

*Frontend*
- [ ] Shopping cart
- [ ] Checkout
- [ ] Totals
- [ ] Payment methods

*Backend*
- [ ] Configuration
- [ ] Order grid
- [ ] Order view
- [ ] Invoice view
- [ ] Credit memo view
- [ ] Shipment view
- [ ] Email sending

*Order Processing (Mollie communication)*
- [ ] Creating the order
- [ ] Invoicing the order
- [ ] Shipping the order
- [ ] Refunding (credit memo) the order

**Other**
If you didn't check any boxes above, please describe your changes in this section.

**Please describe the bug/feature/etc this PR contains:**

When i...

**Scenario to test this code:**

Open the environment and...

### CODE REVIEW

File: Config.php [hunk 1]
```diff
@@ -56,9 +56,10 @@ public function __construct(
     /**
      * @param $path
      * @param null|int|string $storeId
+     * @param string $scope
      * @return string
      */
-    private function getPath($path, $storeId)
+    private function getPath($path, $storeId, $scope = ScopeInterface::SCOPE_STORE)
     {
         return $this->config->getValue($path, ScopeInterface::SCOPE_STORE, $storeId);
```
Comment (DavidLambauer) [inline Config.php:64]:
```suggestion
        return $this->config->getValue($path, $scope, $storeId);
```

Comment (michielgerritsen) [reply Config.php:64]: 🤦

**Comment by Ylmzef** [comment]
It doesn't work for me.

**Comment by Frank-Magmodules** [comment]
Hi @Ylmzef , it seems that your issue not relevant to this issue. Please run `setup:upgrade` to make sure that you're database schema is updated. If that doesn't work please create a new issue which describes the exact issue you're experiencing.

**Pull request merged**